### PR TITLE
Prepare new package naming for Combine

### PR DESCRIPTION
**Pull request by peterfriese**
This change reflects the discussion in b/191035457

## Changes in this PR
* No umbrella package
* Packages are named `Firebase<Product>Combine-Community`
* Targets are named `Firebase<Product>CombineSwift`, to allow `import Firebase<Product>CombineSwift`

## To do before merging
* [ ] comment out all the packages

### CODE REVIEW

**Comment by paulb777** [comment]
LGTM, merging should be coordinated with a decision about a specific release along with any associated materials like blogs and release notes.

**Comment by paulb777** [comment]
Let's get this out to the community and get some more feedback. Thanks!

Please add a README link to docs either in this PR or another.

**Review by paulb777 — APPROVED** [review]

**Comment by peterfriese** [comment]
Updated packages names, sample app, README, installation instructions, and changelog.

File: FirebaseCombineSwift/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-# Unreleased
-
+# v8.8.0
```
Comment (paulb777) [inline FirebaseCombineSwift/CHANGELOG.md:1]: 8.9.0

**Review by paulb777 — APPROVED** [review]
Thanks, please also make sure CI is clean before merging.

**Pull request merged**